vmdb/app/models/host_openstack_infra.rb

```diff
@@ -1,2 +1,5 @@
 class HostOpenstackInfra < Host
+  def self.hostname_required?
+    false
+  end
```

Comment:
Oh wait, this is Host...I was thinking EMS...

@jrafanie Is it even really required on the vmware Host object?  :confused: